app/policies/participation_policy.rb

```diff
@@ -0,0 +1,25 @@
+class ParticipationPolicy < ApplicationPolicy
+  def create?
+    user&.admin?
```

Comment:
So what happens when someone tries to create a participation that doesn't have permission to do so? Is an error passed from the API to the user?